osquery/tables/utility/file.cpp

```diff
@@ -97,6 +97,17 @@ void genFileInfo(const fs::path& path,
     r["type"] = "unknown";
   }
 
+#if defined(__APPLE__)
```

Comment:
My recommendation is to drop this, instead gate this logic with `isPlatform(PlatformType::TYPE_OSX)`

Reply:
I'd like to, but we get an error on non-darwin platforms:
```console
osquery/tables/utility/file.cpp:103:69: error: no member named 'st_flags' in 'stat'
    if (!describeBSDFileFlags(bsd_file_flags_description, file_stat.st_flags)) {
                                                          ~~~~~~~~~ ^
```